core/src/ConfirmationModal.html

```diff
@@ -21,15 +21,19 @@ <h1 class="fd-modal__title">{title}</h1>
 <script type="text/javascript">
   export default {
     data() {
-      //fallback data in case title or text props weren't provided
+      // fallback data in case not provided
```

Comment:
So we have a confirmation modal component. We are using it currently once in App.html, in this case, we are alway providing the data to the component. Whenever you provide the data, it will never use the fallback data, even if the data you provide is undefined. I also tried with single variables instead of an object. If you give variables as input when using the component, it will never take the default values. 
Default values would be taken if we use somewhere else the component and we do not give the 'content' as input. We could make it more complicated to match what you (and mostly anyone) would expect. Let´s discuss the best solution offline.

Reply:
Added mechanism to set default values if not provided. If values are provided, even with a falsy value like empty string or undefined, they will be taken